coala_quickstart/generation/Settings.py

```diff
@@ -48,21 +49,24 @@ def generate_ignore_field(project_dir,
     :param extset:
         A dict with language name as key and a set of extensions as
         value. This includes only those extensions used by the project.
+    :ignore_globs:
+        generator object get_gitignore_glob
```

Comment:
If we are going to follow the suggestion in https://github.com/coala/coala-quickstart/pull/209/files#r158484635, please do create a newcomer issue about this, sometimes we have to intentionally leave some minor corrections for newcomers ;)

Otherwise, change it to something like "Generator/List object containing file globs to ignore